modules/ROOT/pages/migration-connectors-json.adoc

```diff
@@ -12,11 +12,14 @@ To migrate transformations involving JSON, MuleSoft recommends that you use Data
 
 In Mule 3, many components required the payload to be a POJO so that its fields could be accessed. In Mule 4, DataWeave can be used as a query language directly on the JSON payload, which completely avoids the need to convert the payload to a POJO as an intermediate step.
 
-For mapping rules that are configured for the `json:json-to-object-transformer`, you can express the rules in DataWeave.
+=== JSON to Object Transformer
 
-In Mule 3:
+For mapping rules that are configured for the `json:json-to-object-transformer`, you can express the rules in DataWeave.
 
-.Mule 3 Example: Mixin used in the Mapper Configuration
+* In Mule 3:
++
+Example: Mixin used in the Mapper Configuration
```

Comment:
Should this be "Mixin Class" instead of "Mixin"? I saw Mixin class referenced later on.